crates/pop-drink/Cargo.toml

```diff
@@ -4,12 +4,23 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+scale-info = { workspace = true, default-features = false, features = [
+ "derive",
+] }
+
 drink.workspace = true
 ink_sandbox.workspace = true
-pop-runtime-devnet.workspace = true
 pallet-contracts.workspace = true
+pop-runtime-devnet.workspace = true
 frame-system.workspace = true
 frame-support.workspace = true
 sp-io.workspace = true
 scale.workspace = true
 pop-api.workspace = true
+pop-primitives.workspace = true
```

Comment:
Can we not just export whatever is required from this crate in the runtime crate?

Reply:
Resolved